assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -3,6 +3,18 @@
     <span>Current log from {{start}}&nbsp;</span>
     <span ng-if="end">to {{end}}</span>
   </div>
+
+  <div row main-axis="end" ng-if="canViewArchive">
+    <form
+      ng-if="canViewArchive"
+      action="{{kibanaAuthUrl}}"
+      method="POST" target="_blank">
```

Comment:
That would be inconsistent with what we are doing elsewhere today and what
we are already doing is what we agreed on with UXD.  Consistency for now,
we can re-evaluate in 3.2

On Wed, Nov 4, 2015 at 9:54 AM, Ben Petersen <notifications@github.com>
wrote:

> In assets/app/views/directives/logs/_log-viewer.html
> <https://github.com/openshift/origin/pull/5663#discussion_r43889077>:
>
> > @@ -3,6 +3,18 @@
> >      <span>Current log from {{start}}&nbsp;</span>
> >      <span ng-if="end">to {{end}}</span>
> >    </div>
> > +
> > +  <div row main-axis="end" ng-if="canViewArchive">
> > +    <form
> > +      ng-if="canViewArchive"
> > +      action="{{kibanaAuthUrl}}"
> > +      method="POST" target="_blank">
>
> I tend to think anytime you navigate to a new app (even though we have
> Kibana somewhat skinned) it should open a new tab... I'd rather drop the
> back link.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5663/files#r43889077>.
>